.coveragerc

```diff
@@ -1,4 +1,6 @@
 [run]
 parallel = True
 data_file = /tmp/${USER}/cov/coverage.db
+
+[report]
```

Comment:
That's ... interesting. While this may be slightly more costly, I doubt it really matters. Essentially, according to my understanding of the documentation, the difference ought to be that my original change doesn't _collect_ coverage metrics for the site-specific Python libraries, whereas your version collects the data but doesn't report it.

Does this actually make a real difference in the run?

Reply:
Corrected.